### DISCUSSION TITLE
Background image on the login screen

### DISCUSSION THREAD

**Discussion by cairoapcampos** [requarks/wiki#6578] [Help / Questions] [answered]
I resized an image to size 1920x1080. Later I added the image to the login screen. But the image is not loaded.

I've set the Wiki to be private. Therefore, guest users cannot view anything. Can you help me ?

![tela](https://github.com/requarks/wiki/assets/21186839/ac6763df-b927-4010-97ea-7d2f7e35562f)

**Comment by P4mes4n** [comment, marked as answer]
The guest group needs the `read: assets` permission on the path where the picture is located.
Best solution is to create a dedicated folder (eg. `public`) and put the picture there.

**Comment by cairoapcampos** [comment]
@P4mes4n I kept "read:assets" enabled.

![2023-07-18_15-52](https://github.com/requarks/wiki/assets/21186839/93ef833d-28da-4148-bc98-9851c5182ae7)

Then under "Page Rules" I enabled "read:assets" and blocked the rest.

![2023-07-18_16-22](https://github.com/requarks/wiki/assets/21186839/79e7669d-5fdc-492c-95e7-00c39a7faa51)

Next, I set the background image.

![2023-07-18_16-23](https://github.com/requarks/wiki/assets/21186839/68611bc2-c665-43f6-9678-7ec6737bc9d2)

After these settings, the image was added to the background of the login screen. Do I need to configure any additional security settings for my private wiki?

**Reply by P4mes4n** [reply]
No, you should be good to go.

**Reply by cairoapcampos** [reply]
OK. Doubt solved. :)
